src/posix.mak

```diff
@@ -384,6 +384,7 @@ endif
 clean:
 	rm -R $(GENERATED)
 	rm -f dmd $(idgen_output)
+	rm -f backend/cdxxx.c
```

Comment:
Hmm, actually, there's already a `optabgen_output` defined. Just have to delete stuff found in it.